src/Commands/ReapplyController.php

```
@@ -0,0 +1,209 @@
+        if (($file !== '' || $this->provider !== '') && !$anyMatched) {
+            $this->stderr('[scaffold] No tracked files matched the given filter.' . PHP_EOL);
+
+            return ExitCode::UNSPECIFIED_ERROR;
+        }
+
+        if ($anyUpdated) {
+            $lock->write(['providers' => $data['providers'], 'files' => $updatedFiles]);
+        }
+
+        return ExitCode::OK;
+    }
```

PHPMD (2.15.0)

[warning] The method actionIndex() has a Cyclomatic Complexity of 25. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)